src/dev.omaremara.bugtracker/dev/omaremara/bugtracker/model/Project.java

```diff
@@ -0,0 +1,17 @@
+package dev.omaremara.bugtracker.model;
+
+public enum Project {
```

Comment:
`Project` is not an enum, just a class with a single attribute called `name` of type `string`,